Header header
int64 M
int64 K
float64 stddev
GMMTMapClusterMsg[] clusters